src/Sulu/Bundle/AdminBundle/Resources/js/components/ItemSelection/itemSelection.scss

```diff
@@ -0,0 +1,28 @@
+@import '../../containers/Application/colors.scss';
+
+$listElementBackgroundColor: $white;
+$listElementBorderBottomColor: $silver;
+$listElementDragBoxShadowColor: rgba($black, .1);
+
+.list {
+    margin: 0;
+    padding: 0;
+}
+
+.list-element {
+    list-style: none;
+    border-bottom: 1px solid $listElementBorderBottomColor;
+
+    &:last-child:not(:first-of-type) {
```

Comment:
That's a rather confusing selector... What does it achieve?

Reply:
the dragged list item will be put into the body. therefore it will be addressed by the `last-child` selector. To prevent that I added the `:not(:first-of-type)` selector